### PR TITLE
📝 Update includes in `docs/es/docs/tutorial/cookie-params.md`

### PR DESCRIPTION
**Pull request by antonyare93**

### CODE REVIEW

File: docs/es/docs/tutorial/cookie-params.md [hunk 1]
```diff
@@ -6,115 +6,15 @@ Puedes definir parámetros de Cookie de la misma manera que defines parámetros
...
-Es preferible utilizar la versión `Annotated` si es posible.
-
-///
-
-```Python hl_lines="3"
-{!> ../../docs_src/cookie_params/tutorial001.py!}
-```
-
-////
+{*> ../../docs_src/cookie_params/tutorial001_an_py310.py hl[3]*}
```
Comment (krishnamadhavan) [inline docs/es/docs/tutorial/cookie-params.md:9]: Can we remove the `>` character?

File: docs/es/docs/tutorial/cookie-params.md [hunk 2]
```diff
@@ -6,115 +6,15 @@ Puedes definir parámetros de Cookie de la misma manera que defines parámetros
...
-Es preferible utilizar la versión `Annotated` si es posible.
-
-///
-
-```Python hl_lines="9"
-{!> ../../docs_src/cookie_params/tutorial001.py!}
-```
-
-////
+{*> ../../docs_src/cookie_params/tutorial001_an_py310.py hl[9]*}
```
Comment (krishnamadhavan) [inline docs/es/docs/tutorial/cookie-params.md:17]: Can we remove the `>` character?

Comment (antonyare93) [reply docs/es/docs/tutorial/cookie-params.md]: Thanks!

**Comment by alejsdev** [comment]
Thank you! @antonyare93 :nerd_face:

**Pull request merged**